src/pages/guides/using/security_general.md

```diff
@@ -46,3 +46,14 @@ All communications to the Runtime cluster are secured by HTTPS. Communications f
 Actions can call any external IP, however only the following ports are allowed:
 
 `21, 22, 25, 53, 80, 123, 143, [200-299], 389, 443, 445, 465, 587, 636, 1433, [2000-2999], 3000, 3306, 4242, 4317, 4343, 5400, 5432, 5671, 5672, 6061, 6062, 6379, 6380, 6651, 8000, 8020, 8080, 8085, 8088, 8089, 8300, 8500, 8600, 9090, 9092, 9093, 9094, [9096-9352], 9354, [10000-20000], 27016, 27017, 27018, 27019, 30303, 50010, 60020`
+
+
+## Using a Content Delivery Network (CDN)
+It is always a good idea to use a CDN in front of your web actions, as it improves security. Many CDNs offer built-in security features such as DDoS protection, and web application firewall (WAF). These features can protect your web actions against DDoS attacks and other security threats.
+
+You could secure your web action using any CDN by following these steps: 
+
+1. Choose a CDN provider: There are several CDN providers available, including Cloudflare, Fastly, Akamai, and others. Choose a provider that meets your needs and sign up for their service.
+2. Configure the CDN: Once you have signed up for the CDN service, configure it to point to Runtime’s domain name: `adobeioruntime.net`. This can usually be done through the CDN provider's web interface or API.
```

Comment:
In the case where a customer is using a subdomain to reach IO Runtime, (`mynamespace.adobeioruntime.net`) we likely want to include that here. Something like:
> _(or `<your-namespace>.adobeioruntime.net` if you use a subdomain to invoke your actions)_